### PR TITLE
Mufaro navbar

### PR DESCRIPTION
**Pull request by MufaroMakiwa**
Adds a nav bar to the editor page. I commented out the sign in page just to be able to see the changes we were making. All the elements are static and will need to be made dynamic once we link them up with the backend

### CODE REVIEW

File: src/main/webapp/components/Editor/Profile.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import './Profile.css'
+
+function Profile(props) {
+  const {name, side, active} = props;
+
+  return (
+    <div className={`Profile_container ${side==="R"? "Profile_flipped": ""}`}>
```
Comment (andiov) [inline src/main/webapp/components/Editor/Profile.js]: Nit: add space before ? and : for the ternary operator.

`condition ? exprIfTrue : exprIfFalse`

File: src/main/webapp/components/Editor/Profile.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+      <div className="Profile_image">
+        <div className={`Profile_activity ${active ? "active": "inactive"} ${side==="R" ? "right": "left"}`}></div>
```
Comment (andiov) [inline src/main/webapp/components/Editor/Profile.js]: Same here

File: .settings/org.eclipse.core.resources.prefs [hunk 3]
```diff
@@ -0,0 +1,3 @@
+eclipse.preferences.version=1
```
Comment (andiov) [inline .settings/org.eclipse.core.resources.prefs:1]: Do you need to add the .prefs files? Or are they specific to each person's setup?

**Review by andiov — COMMENTED** [review]
Make sure you remove all backup files (ending in ~).

**Review by michaellawes — APPROVED** [review]

File: src/main/webapp/components/Editor/Profile.js (hunk 1 above)
Comment (MufaroMakiwa) [reply src/main/webapp/components/Editor/Profile.js]: Okay thank you. I will fix it.

File: .settings/org.eclipse.core.resources.prefs (hunk 3 above)
Comment (MufaroMakiwa) [reply .settings/org.eclipse.core.resources.prefs:1]: I think some of these files are auto generated so we will need to add them to the gitignore file

**Review by cenofe — APPROVED** [review]

**Pull request merged**